package.json

```
@@ -45,7 +45,7 @@
     "@nextcloud/browserslist-config": "^3.0.1",
     "@nextcloud/calendar-availability-vue": "^2.2.2",
     "@nextcloud/capabilities": "^1.2.0",
-    "@nextcloud/dialogs": "^5.3.5",
+    "@nextcloud/dialogs": "5.3.2",
```

Can be unpinned now, but use 5.3.6 or later

```suggestion
    "@nextcloud/dialogs": "^5.3.6",
```